main.py

```diff
@@ -79,6 +79,16 @@ async def IsMemberWorking(ctx, discorduser, date = datetime.today().strftime('%d
 
     await ctx.send(content = "Done", delete_after = 0.1)
 
+@slash.slash(name = "GetEndOfMonthReport", description = "Calculates the salary deduction percentage for the provided month", options = UI.CreateGetEndOfMonthReportOptions(), guild_ids = guild_ids)
```

Comment:
doesn't only calculate the salary deduction, just say it gets the monthly report

Reply:
Done